hack/make-rules/test-e2e-node.sh

```diff
@@ -129,13 +130,26 @@ if [ $remote = true ] ; then
   echo "Ginkgo Flags: $ginkgoflags"
   echo "Instance Metadata: $metadata"
   # Invoke the runner
-  go run test/e2e_node/runner/run_e2e.go  --logtostderr --vmodule=*=4 --ssh-env="gce" \
-    --zone="$zone" --project="$project"  \
-    --hosts="$hosts" --images="$images" --cleanup="$cleanup" \
-    --results-dir="$artifacts" --ginkgo-flags="$ginkgoflags" \
-    --image-project="$image_project" --instance-name-prefix="$instance_prefix" --setup-node="true" \
-    --delete-instances="$delete_instances" --test_args="$test_args" --instance-metadata="$metadata"
-  exit $?
+  if [[ $gubernator = false || $remote = false ]]; then
+    go run test/e2e_node/runner/run_e2e.go  --logtostderr --vmodule=*=2 --ssh-env="gce" \
+      --zone="$zone" --project="$project" --gubernator="$gubernator" \
+      --hosts="$hosts" --images="$images" --cleanup="$cleanup" \
+      --results-dir="$artifacts" --ginkgo-flags="$ginkgoflags" \
+      --image-project="$image_project" --instance-name-prefix="$instance_prefix" --setup-node="true" \
+      --delete-instances="$delete_instances" --test_args="$test_args" --instance-metadata="$metadata"
+    exit $?
+  else
+    echo "GBR flag on"
+    go run test/e2e_node/runner/run_e2e.go  --logtostderr --vmodule=*=2 --ssh-env="gce" \
+      --zone="$zone" --project="$project" --gubernator="$gubernator" \
+      --hosts="$hosts" --images="$images" --cleanup="$cleanup" \
+      --results-dir="$artifacts" --ginkgo-flags="$ginkgoflags" \
+      --image-project="$image_project" --instance-name-prefix="$instance_prefix" --setup-node="true" \
+      --delete-instances="$delete_instances" --test_args="$test_args" --instance-metadata="$metadata" \
+      2>&1 | tee build-log.txt
+    ./test/e2e_node/g8r.sh
```

Comment:
I call it here for successful tests, and when the test fails, in /kubernetes/test/e2e_node/runner/run_e2e.go, because when it fails the script is not run from test-e2e-node.sh. I'll try to see if I can just call it from run_e2e.go instead of test-e2e-node.sh unless there's some reason I shouldn't.